modules/node/src/test/store.ts

```diff
@@ -0,0 +1,54 @@
+import { CFCoreTypes, StorePair } from "@connext/types";
+
+export class MemoryStoreService implements CFCoreTypes.IStoreService {
```

Comment:
Oooh another copy/paste of the in-memory store. Here's a little extra incentive to DRY out the store code by moving it into it's own module ASAP

Reply:
Yep, this one is deleted though. I'll do that though.